lib/api/pulsar/pulsar_topic_api.dart

```diff
@@ -74,6 +74,20 @@ class PulsarTopicApi {
     return jsonResponse.map((name) => TopicResp.fromJson(name)).toList();
   }
 
+  static Future<int> getTopicPartitionedNum(
```

Comment:
can we use `getBase` in `pulsar_partitioned_topic_api.dart`?

Reply:
yes